packages/white-whale-std/src/pool_manager.rs

```diff
@@ -246,3 +239,14 @@ pub enum QueryMsg {
     #[returns(PairInfo)]
     Pair { pair_identifier: String },
 }
+
+/// The response for the `AssetDecimals` query.
+#[cw_serde]
+pub struct AssetDecimalsResponse {
+    /// The pair identifier to do the query for.
+    pub pair_identifier: String,
+    /// The queried denom in the given pair_identifier.
+    pub denom: String,
+    /// The decimals for the requested denom.
+    pub decimals: u8,
+}
```

Comment:
We should be able to delete the old `NativeTokenDecimalsResponse` as well.

Reply:
Well the old `NativeTokenDecimalsResponse` is being used on the V1 contracts, so let it be for now.